STT/Dockerfile.train

```diff
@@ -0,0 +1,249 @@
+ARG kenlm_repo=kpu/kenlm
+ARG kenlm_branch=87e85e66c99ceff1fab2500a7c60c01da7315eec
+
+# Model parameters
+ARG model_language=fr
+ENV MODEL_LANGUAGE=$model_language
+
+# Training hyper-parameters
+ARG train_batch_size=64
+ENV TRAIN_BATCH_SIZE=$train_batch_size
+
+ARG dev_batch_size=64
+ENV DEV_BATCH_SIZE=$dev_batch_size
+
+ARG test_batch_size=64
+ENV TEST_BATCH_SIZE=$test_batch_size
+
+ARG n_hidden=2048
+ENV N_HIDDEN=$n_hidden
+
+ARG epochs=40
+ENV EPOCHS=$epochs
+
+ARG learning_rate=0.0001
+ENV LEARNING_RATE=$learning_rate
+
+ARG dropout=0.3
+ENV DROPOUT=$dropout
+
+ARG lm_top_k=500000
+ENV LM_TOP_K=500000
+
+ARG lm_alpha=0.0
+ENV LM_ALPHA=$lm_alpha
+
+ARG lm_beta=0.0
+ENV LM_BETA=$lm_beta
+
+ARG beam_width=500
+ENV BEAM_WIDTH=$beam_width
+
+ARG early_stop=1
+ENV EARLY_STOP=$early_stop
+
+ARG amp=0
+ENV AMP=$amp
+
+# Skipping batch test to avoid hanging processes
+# Should be set to 0 by default once STT#2195 is fixed
+# See https://github.com/coqui-ai/STT/issues/2195 for more details
+ARG skip_batch_test=1
+ENV SKIP_BATCH_TEST=$skip_batch_test
+
+# Dataset management
+ARG duplicate_sentence_count=1
+ENV DUPLICATE_SENTENCE_COUNT=$duplicate_sentence_count
+
+# Should be of the form: lm_alpha_max,lm_beta_max,n_trials
+ARG lm_evaluate_range=
+ENV LM_EVALUATE_RANGE=$lm_evaluate_range
+
+# Data augmentation
+ARG enable_augments=0
+ENV ENABLE_AUGMENTS=$enable_augments
+
+# Others
+ARG english_compatible=0
+ENV ENGLISH_COMPATIBLE=$english_compatible
+
+ARG lm_add_excluded_max_sec=0
+ENV LM_ADD_EXCLUDED_MAX_SEC=$lm_add_excluded_max_sec
+
+# To fine-tune using your own data
+ARG cv_personal_first_url=
+ENV CV_PERSONAL_FIRST_URL=$cv_personal_first_url
+
+ARG cv_personal_second_url=
+ENV CV_PERSONAL_SECOND_URL=$cv_personal_second_url
+
+ARG log_level=1
+ENV LOG_LEVEL=$log_level
+
+ARG uid=999
+ENV UID=$uid
+
+ARG gid=999
+ENV GID=$gid
+
+# Make sure we can extract filenames with UTF-8 chars
+ENV LANG=C.UTF-8
+
+# Avoid keyboard-configuration step
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV HOMEDIR /home/trainer
+
+ENV VIRTUAL_ENV_NAME stt-train
+ENV VIRTUAL_ENV $HOMEDIR/$VIRTUAL_ENV_NAME
+ENV STT_DIR $HOMEDIR/stt
+ENV CC_DIR $HOMEDIR/cc
+
+ENV STT_BRANCH=$stt_branch
+ENV STT_SHA1=$stt_sha1
+
+ENV PATH="$VIRTUAL_ENV/bin:${HOMEDIR}/tf-venv/bin:$PATH"
+
+RUN env
+
+# Get basic packages
+RUN apt-get -qq update && apt-get -qq install -y --no-install-recommends \
+    build-essential \
+    curl \
+    wget \
+    git \
+    python3 \
+    python3-pip \
+    ca-certificates \
+    cmake \
+    libboost-all-dev \
+    zlib1g-dev \
+    libbz2-dev \
+    liblzma-dev \
+    libmagic-dev \
+    libopus0 \
+    libopusfile0 \
+    libsndfile1 \
+    libeigen3-dev \
+    pkg-config \
+    g++ \
+    python3-venv \
+    unzip \
+    pixz \
+    sox \
+    sudo \
+    libsox-fmt-all \
+    ffmpeg \
+    locales locales-all \
+    xz-utils \
+    software-properties-common
+
+# For exporting using TFLite
+RUN add-apt-repository ppa:deadsnakes/ppa -y
```

Comment:
what is this adding? python3.7 ?

Reply:
`/tflite-venv/` to export using TFLite.
https://github.com/coqui-ai/STT/pull/2221